docs/fr/2-Publication/R-Shiny.md

```diff
@@ -1,40 +1,120 @@
 # Aperçu
 
-R-Shiny est un paquet R qui facilite la création d'applications Web interactives
-dans R. Nous gérons le serveur R-Shiny, et il est très facile de mettre votre
-tableau de bord sur le Plate-forme.
+R-Shiny est un package R qui facilite la création d'applications Web interactives dans R.
 
-![Page d'accueil Shiny](../images/readme/shiny_ui.png)
+<!-- prettier-ignore -->
+!!! info "Hébergement d'applications R Shiny"
+     Nous ne prenons actuellement pas en charge l'hébergement d'applications R Shiny, mais vous pouvez les créer. Nous souhaitons activer l'hébergement de l'application R Shiny à l'avenir.
+
+![Page d'accueil brillante](../images/readme/shiny_ui.png)
+
+## R brillant
+
+_Publier des graphismes de qualité professionnelle_
+
+[![InteractiveDashboard](../images/InteractiveDashboard.PNG)](/2-Publishing/R-Shiny/)
+
+R Shiny est un framework d'application Web open source qui permet aux data scientists et aux analystes de créer des tableaux de bord et des visualisations de données interactifs basés sur le Web à l'aide du langage de programmation R. L'un des principaux avantages de R Shiny est qu'il offre un moyen simple de créer des tableaux de bord interactifs de haute qualité sans avoir besoin d'une expertise approfondie en développement Web. Avec R Shiny, les scientifiques des données peuvent tirer parti de leurs compétences en codage R pour créer des applications Web dynamiques, basées sur les données, qui peuvent être facilement partagées avec les parties prenantes.
```

Comment:
Change "scientifiques des données" to data scientists
Change "parties prenantes" to acteurs